src/components/Section/Identification/Identification.jsx

```diff
@@ -14,8 +15,74 @@ import OtherNames from './OtherNames'
 import Physical from './Physical'
 import ContactInformation from './ContactInformation'
 
+const storeToComponentMap = {
+  ApplicantBirthDate,
+  ApplicantBirthPlace,
+  ApplicantName,
+  ApplicantSSN,
+  Contacts: ContactInformation,
+  OtherNames,
+  Physical
+}
+
 class Identification extends SectionElement {
+  getSubsectionComponent (name) {
+    // https://reactjs.org/docs/jsx-in-depth.html#choosing-the-type-at-runtime
+    const SubsectionComponent = storeToComponentMap[name]
+    if (!SubsectionComponent) {
+      console.error(`${name} component not found`)
+    }
+    return SubsectionComponent
+  }
+
+  // Returns the component corresponding to the provided subsection object
+  createSubsection (subsection) {
+    const SubsectionComponent = this.getSubsectionComponent(subsection.store)
+
+    const props = {
+      ...this.props[subsection.store],
+      name: subsection.url,
+      dispatch: this.props.dispatch,
+      onUpdate: this.handleUpdate.bind(this, subsection.store),
+      onError: this.handleError
+    }
+
+    return React.createElement(SubsectionComponent, props)
+  }
+
+  // Returns an array of SectionViews with their corresponding child component, based on the navigation
+  createSectionViews () {
+    const section = 'identification'
+    const sectionNav = navigation.find(n => n.url === section)
+    const subsections = sectionNav.subsections
+
+    const views = subsections.map((subsection, i) => {
+      if (subsection.exclude) {
```

Comment:
The `exclude` flag comes from [here](https://github.com/18F/e-QIP-prototype/blob/83c476752d6bc1e6d55dfcf0a5056f2a0b3e42a1/src/config/navigation.js#L13). This conditional is excluding the Introduction and Review sections from this generated list, since that's still rendered inline [below](https://github.com/18F/e-QIP-prototype/blob/03eeabb7f7267d0d474030265b310abbd5c84088/src/components/Section/Identification/Identification.jsx#L89-L98). Might move it in here at some point, but will require some logic specific to those subsections, as they render a bit differently.

Reply:
The `exclude` flag existed before right? Do you know what its purpose is generally?

Reply:
It did, yes. It's used for some navigation stuff.

* https://github.com/18F/e-QIP-prototype/blob/36635853dd120a5905db82f6b53217dd50ee2a8e/src/components/Navigation/navigation-helpers.js#L3-L14
* https://github.com/18F/e-QIP-prototype/blob/36635853dd120a5905db82f6b53217dd50ee2a8e/src/components/Navigation/navigation-helpers.js#L123-L125

Might consider renaming `exclude` to make it more clear ("Exclude" from what?) - or do these things another way.